toolium/behave/env_utils.py

```diff
@@ -130,8 +130,7 @@ def __init__(self, **kwargs):
         self.logger = Logger(logger_class, self.show)
         self.init_actions()
         self.scenario_counter = 0
-        self.feature_error = False
-        self.scenario_error = False
```

Comment:
These two variables are used to execute the `Actions After the Feature` actions in case of error in Scenario actions

Reply:
You are right. It is interesting to run 'after' hooks if something fails. Reverting this change.